features/input-date-time.yml

```diff
@@ -0,0 +1,16 @@
+name: <input> date, datetime-local, time, and week
```

Comment:
Should find a way to construct the title that is consistent across the single and multiple types. The multiple-types version should also be clear that these are values of the `type` attribute, like the single-type version does.